scripts/check_upgrade_maps.py

```diff
@@ -13,12 +13,13 @@
 import json
 import subprocess
 
+nix_flags = ['--extra-experimental-features', 'nix-command', '--extra-experimental-features', 'flakes', '--extra-experimental-features', 'discard-references']
```

Comment:
this can be `['--extra-experimental-features', 'nix-command flakes discard-references']` or `['--extra-experimental-features', '"nix-command flakes discard-references"']` (depending on how the command is run - the features just need to be in the same argv entry)

Reply:
:+1: